client/client_update_test.go

```diff
@@ -893,6 +965,53 @@ func TestUpdateNonRootRemoteCorruptedNoLocalCache(t *testing.T) {
 				role: role,
 			}, testData, true)
 		}
+
+		switch role {
+		case data.CanonicalSnapshotRole:
+			for _, missing := range []string{data.CanonicalRootRole, data.CanonicalTargetsRole} {
+				testData := swizzleExpectations{
+					desc: fmt.Sprintf("snapshot missing root meta checksum"),
+					// TODO: this should probably be always client.ErrMissingMeta
```

Comment:
Out of curiosity, where is this json syntax error originating from?  It doesn't seem like this should affect the structural integrity of `data.Signed`, unless I'm missing something?